package.json

```diff
@@ -52,7 +54,8 @@
     "metro-react-native-babel-preset": "^0.59.0",
     "prettier": "^2.0.2",
     "react": "16.9.0",
-    "react-native": "0.61.5",
+    "react-native": "0.62",
+    "react-native-windows": "0.62",
```

Comment:
We also need to declare a peer dependency on React Native Windows, or we will otherwise not see this work outside of the sample app in certain configurations. That should be in a similar form to the existing RN peer dependency.